OPHD/ophd.vcxproj

```diff
@@ -135,7 +135,7 @@ IF NOT "$(VcpkgInstalledDir)" == "" (
       <AdditionalOptions>/we4242 /we4254 /we4263 /we4265 /we4287 /we4289 /we4296 /we4311 /we4545 /we4546 /we4547 /we4549 /we4555 /we4619 /we4640 /we4826 /we4905 /we4906 /we4928 %(AdditionalOptions)</AdditionalOptions>
     </ClCompile>
     <Link>
-      <AdditionalDependencies>NAS2D.lib;opengl32.lib;sdl2maind.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>NAS2D.lib;opengl32.lib;sdl2main.lib;%(AdditionalDependencies)</AdditionalDependencies>

```

Comment:
Btw, the "d" at the end is for the Debug version of the library.

I should point out this config is for "Debug|x64".

Reply:
Oh I know. vcpkg changes something that broke the build in debug mode so I had to change it.